src/lib/pixel-select-drop-down/pixel-select-drop-down.tsx

```diff
@@ -14,11 +14,15 @@ export interface DropDownProps extends InputHTMLAttributes<HTMLDivElement> {
   isShowLabel?: boolean
   disabled?: boolean
   customLabel: string
+  isShowCheckbox?: boolean
+  isShowRadio?: boolean
+  onSelectedOptions: any
 }
 interface OptionsData {
   value: string
   label: string
   disabled?: boolean
+  checked: boolean[]
```

Comment:
remove `[]` operator 
`boolean[]` means that you are sending data in this form `[True, False, True, True, False]
 `